docs/Makefile

```diff
@@ -26,6 +26,8 @@ clean:
 	rm -f errors.txt
 	rm -rf images/auto-generated
 	rm -f getting-started/external_examples.rst
+	find . -type f -name "_autosummary" -delete
+	find . -type f -name "api-autogen" -delete
```

Comment:
Do these match anything? `_autosummary` and `_api-autogen` look like directories to me, but `-type f` matches files.

Reply:
My mistake.  In fact, we only need `_autosummary`.